stacks-core/src/codec.rs

```diff
@@ -0,0 +1,52 @@
+/*!
+Module for contract name parsing
```

Comment:
Is codec really for contract name parsing? I thought that was the contract_name.rs module. I assume this is a copy paste error.

Reply:
Yep not sure how that happened. It's fixed now.